tools/qontract_cli.py

```diff
@@ -2269,6 +2270,75 @@ def slo_document_services(ctx, status_board_instance):
     print_output(ctx.obj["options"], slodocs, columns)
 
 
+@get.command()
+@click.pass_context
+def network_connections(ctx):
+    import reconcile.terraform_tgw_attachments as tftgw
+
+    desired_state_data_source = tftgw._fetch_desired_state_data_source()
+    accounts = [a.dict(by_alias=True) for a in desired_state_data_source.accounts]
+
+    vault_settings = get_app_interface_vault_settings()
+    secret_reader = create_secret_reader(vault_settings.vault)
+    aws_api = AWSApi(1, accounts, secret_reader=secret_reader, init_users=False)
+
+    ocm_map = tftgw._build_ocm_map(desired_state_data_source.clusters, vault_settings)
+    desired_state, err = tftgw._build_desired_state_tgw_attachments(
+        desired_state_data_source.clusters,
+        ocm_map,
+        aws_api,
+    )
+    if err:
+        raise RuntimeError("Could not find VPC ID for cluster")
+
+    tftgw._validate_tgw_connection_names(desired_state)
+
+    print("graph LR")
+    for item in desired_state:
+        r = item.requester
+        source = f"{r.account.name}/{r.region}/{r.tgw_id}"
+        a = item.accepter
+        target = f"{a.account.name}/{a.region}/{a.vpc_id}"
+        print(f"    {source} --> {target}")
```

Comment:
There is a flag field `deleted` in `item`, real integration run will skip it https://github.com/app-sre/qontract-reconcile/blob/393123ac83be16c432faa5c92832c0e5d98f6b2d/reconcile/utils/terrascript_aws_client.py#L1090

Can either skip it here or render as

```mermaid
graph LR
A --> |deleted| B
```

similarly for vpc peerings.

Reply:
thanks! https://github.com/app-sre/qontract-reconcile/pull/3914/commits/dd6e4c7376b0d36077383cbb446ba7d1ee3efe76